spec/features/hpv_vaccination_triage_during_vaccination_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+RSpec.describe "HPV Vaccination" do
```

Comment:
Not sure whether this matches the new scope of the test. 

Also the spec name has vaccination in it twice; maybe `spec/features/hpv_nurse_triage_during_vaccination_spec.rb`

Reply:
Ok, taken another stab at naming ...